style.css

```diff
@@ -92,10 +94,208 @@ nav img:hover {
+  margin-top: 20px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.content1 h2 {
+  width: 327px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 30px;
+  align-items: center;
+  color: #091e42;
+}
+
+.content1 p {
+  width: 327px;
+  height: 147px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 20px;
+  color: #091e42;
+}
+
+.content1 div {
+  display: flex;
+  align-items: center;
+  padding: 10px 12px;
+  width: 49px;
+  height: 40px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+}
+
+.content1 div h3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 61px;
+  height: 40px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+}
+
+.content1 button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  font-size: 15px;
+  width: 124px;
+  height: 48px;
+  color: #fff;
+  background-color: #ff6b00;
+}
+
+.cards {
+  background-image: url(Images/Img\ Placeholderbackground\ image.png);
+  background-size: cover;
+  background-position: center;
+}
+
+.cards h1 {
+  width: 295px;
+  height: 81px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.cards p {
+  width: 295px;
+  height: 87px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.cards div {
+  display: flex;
+}
+
+.cards div h3 {
+  width: 71px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.cards button {
+  width: 327px;
+  height: 48px;
+  background-color: #ff6b00;
+}
+
+.About {
+  width: 375px;
+  padding: 0 30px;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
+
+.About div h1 {
+  width: 327px;
+  height: 79px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.About div p {
+  width: 327px;
+  height: 148px;
   font-style: normal;
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
+  color: #f4f5f7;
+}
+
+.About div button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 161px;
+  height: 48px;
+  font-size: 15px;
+  background-color: #ff6b00;
+}
+
+.About div h2 {
+  width: 164px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
   color: #fff;
 }
+
+.About div h3 {
+  width: 114px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.properties {
+  text-align: center;
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the margin )top, bottom, left, right) defined in the design between the each project  and on the "About me" section kindly make sure that it has the proper background image (decorations are always back-grounded images, not `img` tags) and for make sure that for technology categories uses the font and colours (background and font) defined for the template.

Your Work section Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/205630366-b6f0919c-a2d8-4f77-9598-3fac817a726e.PNG)

The Figma design Work section Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205630574-6ff16b9a-ba43-4fbb-bd78-a21e2d333c2c.PNG)


Your About section look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/205630930-35fdca47-3e06-47f7-a0d4-a8cc65e1a84e.PNG)

The Figma design About section Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/205631324-e285bf17-4cfd-4052-8c1c-16999a814276.PNG)